gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleProject.java

```diff
@@ -242,4 +243,23 @@ private GradleRunConfigurationAttributes getRunConfigurationAttributes( String t
             showExecutionView, showConsoleView );
     }
 
+    @Override
+    public <T> T adapt( Class<T> adapterType )
```

Comment:
We don't need implementation of DSCOmponentPRovider to be provied from gradle.

The creation of DSCOmponents is not specific to gradle or maven.  It can be done completely inside of project.core.

We will not be fixing dependencies as a part of this for now.  So lets not handle modifiying dependencies.

Lets simplfy this to just creating the .java files.